contracts/ERC20.sol

```diff
@@ -1,16 +1,14 @@
 pragma solidity ^0.4.11;
 
-
 import './ERC20Basic.sol';
 
-
 /**
  * @title ERC20 interface
- * @dev see https://github.com/ethereum/EIPs/issues/20
+ * @dev Implements ERC20 Token Standard: https://github.com/ethereum/EIPs/issues/20
  */
 contract ERC20 is ERC20Basic {
-  function allowance(address owner, address spender) constant returns (uint256);
-  function transferFrom(address from, address to, uint256 value) returns (bool);
-  function approve(address spender, uint256 value) returns (bool);
-  event Approval(address indexed owner, address indexed spender, uint256 value);
+	function transferFrom(address _from, address _to, uint256 _value) returns (bool success);
+	function approve(address _spender, uint256 _value) returns (bool success);
+	function allowance(address _owner, address _spender) constant returns (uint256 remaining);
+	event Approval(address indexed _owner, address indexed _spender, uint256 _value);
```

Comment:
Please use space instead of tabs, because tabs show up differently in different editors, but spaces don't.

Reply:
gotcha, didn't know that. done. btw, I've been changing it to 4 spaces instead of 2 as recommended by Solidity docs.